crates/rpc/types/src/types.rs

```diff
@@ -83,7 +83,7 @@ pub struct RpcL1Status {
     pub last_published_txid: Option<Txid>,
 
     /// number of published transactions in current run (commit + reveal pair count as 1)
-    pub published_inscription_count: u64,
+    pub published_envelope_count: u64,
```

Comment:
Make sure this is correct with respect to envelopes vs envelope bundles or rename it to mean what we want.  Also update doc comment.

Reply:
As of this PR, a single commit reveal pair contains a single envelope, so this should be fine. Will change accordingly if needed.

Reply:
We agreed that the commit-reveal assembly logic in the call today will still handle multiple envelopes, so it's not precise.  Being loose with language in scenarios like this causes chaos when we refactor later.

Reply:
Updated to be published_reveal_txs_count.

Reply:
> Updated to be published_reveal_txs_count.

I don't it. I still see it as `published_envelope_count`

Reply:
@storopoli A reveal tx with multiple envelopes won't correctly result this counter to be incremented by the correct amount.

Reply:
Yes, we need separate counter for the envelopes.